tests/groupTree.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import sunbeam
+import datetimen
+
+spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+
+
+class TestGroupTree(unittest.TestCase):
+    spe3 = None
```

Comment:
Instead of initializing `spe3` class attribute to `None` you could use the `setUpClass` method:

```python
@classmethod
def setUpClass(cls):
     cls.spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')


def test_grouptree(self):
     schedule = self.spe3.schedule
     group_tree = schedule.groupTree( report_step )
```